### PR TITLE
[TED] Update and simplify ruleset

### PR DESCRIPTION
**Pull request by Bisaloo**
Even though I removed the wildcards, the coverage is expanded. All previously secured targets are still secured after my changes.

### CODE REVIEW

**Comment by Bisaloo** [comment]
I didn't review all subdomains from `Sublist3r` but instead browsed the source code to find subdomains currently in use.

File: src/chrome/content/rules/TED.xml [hunk 1]
```diff
@@ -1,60 +1,73 @@
...
+	<target host="hls.ted.com" />
+		<test url="http://hls.ted.com/talks/2806.json" />
+	<target host="images-ssl.ted.com" />
+		<test url="http://images-ssl.ted.com/assets/conferences/TED2016/images/T16_homepage_main.jpg" />
+	<target host="support.ted.com" />
+	<target host="ted2017.ted.com" />
+	<target host="ted2018.ted.com" />
+	<target host="tedglobal2017.ted.com" />
+	<target host="tedlive.ted.com" />
+	<target host="tedwomen2017.ted.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/TED.xml]: conferences.ted.com
fellowsblog.ted.com
goto.ted.com
hello.ted.com
reproxy.ted.com
tedwomen2016.ted.com
tedxbillion.ted.com
tedxinnovations.ted.com
tedxtalks.ted.com

File: src/chrome/content/rules/TED.xml [hunk 2]
```diff
@@ -1,60 +1,73 @@
 <!--
-	For problematic rules, see TED-mismatches.xml.
+	Invalid certificate:
+		images.ted.com
+			<test url="https://images.ted.com/assets/conferences/TED2016/images/T16_homepage_main.jpg" />
```
Comment (J0WI) [inline src/chrome/content/rules/TED.xml]: t.ted.com

File: src/chrome/content/rules/TED.xml [hunk 3]
```diff
@@ -1,60 +1,73 @@
...
-		- conferences.ted.com	(shows default RHEL page)
-		- images.ted.com *
-		- media.ted.com *
-
-	* Times out
+	Time out:
+		video.ted.com
 
 -->
 <ruleset name="TED (buggy)" >
```
Comment (J0WI) [inline src/chrome/content/rules/TED.xml]: I think we can remove the `buggy` flag.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**